packaging/ios/build.sh

```diff
@@ -242,69 +236,15 @@ echo next building for $ARCH
 			-DCMAKE_PREFIX_PATH=${PREFIX} \
 			-DCURL=OFF \
 			-DUSE_SSH=OFF \
-			../libgit2-${LIBGIT2_VERSION}/
-		sed -i.bak 's/C_FLAGS = /C_FLAGS = -Wno-nullability-completeness -Wno-expansion-to-defined /' CMakeFiles/git2.dir/flags.make
+			../libgit2/
+#TEMP		sed -i.bak 's/C_FLAGS = /C_FLAGS = -Wno-nullability-completeness -Wno-expansion-to-defined /' CMakeFiles/git2.dir/flags.make
```

Comment:
Not sure why you did that, but it breaks the Travis build because without removing those flags the build log exceeds 4MB